src/sbvr-api/permissions.ts

```diff
@@ -387,12 +392,60 @@ const $checkPermissions = (
 	});
 };
 
-const constrainedPermissionError = new PermissionError();
-const generateConstrainedAbstractSql = (
+const convertToLambda = (filter: AnyObject, identifier: string) => {
```

Comment:
I think renaming to `lamdaAlias` would make the code a little clearer
```suggestion
const convertToLambda = (filter: AnyObject, lambdaAlias: string) => {
```